manifests/init.pp

```diff
@@ -72,10 +73,14 @@
   }
 
   file { $dhcp_dir:
-    mode    => '0755',
+    group   => $dhcp_root_group,
     require => Package[$packagename],
   }
 
+  if $conf_dir_mode {
+    File[$dhcp_dir] { mode => $conf_dir_mode }
```

Comment:
You can pass an undef value. It'll use the default value

Reply:
:+1: done